JUnit version 4.12
.E

Suppositions faites :
 - procuration = class allumettes.JeuProxy
 - jeuReel     = class allumettes.JeuAllumettes


Time: 0,032
There was 1 failure:
1) testerRespectArchitecture(FormeC7Test)
java.lang.AssertionError: MÃ©thode manquante dans la classe allumettes.Arbitre : arbitrer(allumettes.Jeu)
	at org.junit.Assert.fail(Assert.java:88)
	at IntrospectionUtil.getMethode(IntrospectionUtil.java:37)
	at FormeC7Test.testerRespectArchitecture(FormeC7Test.java:39)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:829)

FAILURES!!!
Tests run: 1,  Failures: 1

